actors/builtin/miner/deadline_state.go

```diff
@@ -1006,7 +1007,7 @@ func (dl *Deadline) RescheduleSectorExpirations(
 			return nil
 		}
 
-		moved, err := partition.RescheduleExpirations(store, sectors, expiration, sectorNos, ssize, quant)
+		moved, replaced, err := partition.RescheduleExpirations(store, sectors, expiration, sectorNos, ssize, quant)
```

Comment:
Moved is just the sector numbers of `replaced`, and is unused except for testing whether it's empty. Remove the return value in favour of the new one you've added.